test/integration/pipeline/tx_pipeline_integration_test.cpp

```diff
@@ -15,6 +15,8 @@
  * limitations under the License.
  */
 
+#include "builders/protobuf/transaction.hpp"
+#include "builders/protobuf/queries.hpp"
 #include "responses.pb.h"
```

Comment:
Doesn't seem ordered, may you please fix it

Reply:
So, should apply clang-format?

Reply:
It is good practice to enable it by-default, so you won't forget it